dspace-api/src/main/java/org/dspace/content/BitstreamServiceImpl.java

```diff
@@ -465,4 +465,16 @@ public int countBitstreamsWithoutPolicy(Context context) throws SQLException {
     public Long getLastModified(Bitstream bitstream) {
         return bitstreamStorageService.getLastModified(bitstream);
     }
+
+    public void setSequenceId(Context context, Bitstream bitstream, Item item, Integer sequenceId) {
```

Comment:
Please add Javadocs to this new Public method.  It's not a simple Setter, and it's an opportunity to better describe a sequence ID.

Also, this implementation looks slightly odd to me.  Must Sequence IDs be *unique across every Bundle*?  That's not something noted in the database level (this column is not set to be unique). For some reason I thought that sequenceID was just unique within a Bundle, but maybe I'm mistaken?

Reply:
@abollini : I admit, the sequence ID is not well documented in the current API code.  I was searching for an answer myself, so it seems like this is an opportunity to better document its purpose (at minimally the REST API layer).

In any case, if both of you feel the Sequence ID should be unique inside the Item, then that's OK with me. We should just document this minimally in JavaDocs (as I've been unable to find any such documentation/JavaDocs, but maybe I'm overlooking it)

Reply:
This overlaps with https://github.com/DSpace/Rest7Contract/pull/68#discussion_r312101033
It's probably best to discuss the sequence ID there instead

Reply:
If we've decided to deprecate sequenceID (per the discussion in the Contract), then we likely should make it read-only here. That'd imply that we'd remove the `setSequenceId()` method altogether, as it shouldn't be necessary.

Reply:
Per [today's meeting](https://wiki.duraspace.org/display/DSPACE/2019-08-22+DSpace+7+Working+Group+Meeting#id-2019-08-22DSpace7WorkingGroupMeeting-Notes), we've decided that sequenceID should be *read-only* in the REST API (for now).  We won't deprecate it yet though (I'll create a JIRA ticket for more analysis on that topic). 

So, this means the `setSequenceId()` method here can be removed (and all related code/tests)